### DISCUSSION TITLE
Online HTML Documentation

### DISCUSSION THREAD

**Discussion by WayneParham** [cc65/cc65#1774] [General]
Not a super-high priority, but can we update the online documentation at https://cc65.github.io/doc/ please?

What is displayed now is a little outdated - we've made corrections and added features.

**Comment by polluks** [comment]
Indeed, should be done automagically.

**Comment by mrdudz** [comment]
Yes, it should happen automatically. However, so far noone came up with a solution and i happen to work on something else right now :)

**Comment by WayneParham** [comment]
No rush at all.  It's very low on the scale of importance, in my opinion.  I just wanted to make a reminder that it's grown stale enough that it might be useful to manually send an updated copy of the docs directory to the webserver.
